Kernel/FileSystem/Ext2FileSystem.h

```diff
@@ -176,7 +176,7 @@ class Ext2FS final : public BlockBasedFileSystem {
         BlockIndex bitmap_block_index { 0 };
         bool dirty { false };
         NonnullOwnPtr<KBuffer> buffer;
-        BitmapView bitmap(u32 blocks_per_group) { return BitmapView { buffer->data(), blocks_per_group }; }
+        Bitmap bitmap(u32 blocks_per_group) { return Bitmap { buffer->data(), blocks_per_group }; }
```

Comment:
I believe this change actually adds extra "unfailable-kmalloc" to the ext2 filesystem.. probably out of scope for this const-correctness change list but probably something to make sure ends up on @bgianfo 's allocation yak stack issue.

Reply:
Oh wait no, change course. this bitmap is non-owning because I calls a *different* two augment constructor. 🤔. Makes me think that changing that bool to an enum and making it required would make it more obvious when a non owning bitmap is intended

Reply:
Correct, this does not allocate ^^

So far, all uses of this constructor are non-owning, and there is not a single place that passes `true`.